examples/basics/custom-type/custom-type.go

```diff
@@ -25,17 +29,24 @@ type (
 )
 
 func main() {
-	// Connect to a local Qlik Associative Engine
+	// Fetch the QCS_HOST and QCS_API_KEY from the environment variables
+	QCS_HOST := os.Getenv("QCS_HOST")
```

Comment:
nit: Usually only constants may be written as all upper-case letters, and, since title case denotes public functions/variables, scoped variables are usually written with a small first letter.

Reply:
✅